apps/dashboard/src/components/create-workflow-button.tsx

```diff
@@ -170,6 +173,7 @@ export const CreateWorkflowButton = (props: CreateWorkflowButtonProps) => {
                     <FormControl>
                       <Textarea
                         placeholder="Description of what this workflow does"
+                        state={errors['description']?.message ? 'error' : 'default'}
```

Comment:
Should we take a similar approach for `Textarea` primitive component as we're doing for the `InputField` [here](https://github.com/novuhq/novu/pull/7044/files#r1848710999)?

Reply:
yes, we can do that and have `Textarea,` which would be used under the form, and have another `TextareaPure` for the cases when we don't have a form

Reply:
That sounds great!